ppr-ui/src/composables/mhrRegistration/home-owner.ts

```diff
@@ -44,6 +49,48 @@ export function useHomeOwners (isPerson: boolean = false, isEditMode: boolean =
     isGlobalEditingMode.value = isEditing
   }
 
+  const getHomeTenancyType = (): HomeTenancyTypes => {
+    const numOfOwners = getMhrRegistrationHomeOwners.value?.length
+
+    if (showGroups.value) {
+      // At leas one group showing with one or more owners
+      return HomeTenancyTypes.COMMON
+    } else if (numOfOwners === 1) {
+      // One owner without groups showing
+      return HomeTenancyTypes.SOLE
+    } else if (numOfOwners > 1) {
+      // More than one owner without groups showing
+      return HomeTenancyTypes.JOINT
+    }
+    return HomeTenancyTypes.NA
+  }
+
+  const getGroupTenancyType = (group: MhrRegistrationHomeOwnerGroupIF): HomeTenancyTypes => {
+    const numOfOwnersInGroup = group.owners.length
+
+    if (numOfOwnersInGroup === 1) {
+      return HomeTenancyTypes.COMMON
+    } else if (numOfOwnersInGroup > 1) {
+      return HomeTenancyTypes.JOINT
+    } else {
+      return HomeTenancyTypes.NA
+    }
+  }
+
+  /**
+   * Get Ownership Allocation status object to conveniently show total allocation
+   * and an allocation error status if exists
+   */
+  const getTotalOwnershipAllocationStatus = (): MhrRegistrationTotalOwnershipAllocationIF => {
+    const totalFractionalNominator = sumBy(getMhrRegistrationHomeOwnerGroups.value, 'interestNumerator')
```

Comment:
Other developers could look up `sumBy` in Lodash but perhaps a small comment here or in the docblock would be helpful in describing whats happening here in particular.

Reply:
Yea sounds good, I will add a comment.